jobs/mysql_backup_job.go

```diff
@@ -96,7 +116,14 @@ func (j *MySQLBackupJob) Run(ctx context.Context, send Send) (rerr error) {
 		}
 	}
 
-	xtrabackupCmd := exec.CommandContext(ctx, xtrabackupBin, "--compress", "--backup") // #nosec G204
+	return nil
+}
+
+func (j *MySQLBackupJob) backup(pipeCtx context.Context) (rerr error) {
```

Comment:
We should wrap pipeline-related commands to function to prevent sending job result message. I.e we need all defers with `cmd.Wait` to finish successfully before sending job result. 

This is important because one failed command will not break whole pipeline, for example `cat xx | echo message` cat will exit with error code, but echo will print `message` and will terminate successfully.